COPY/usr/bin/generate_miq_server_cert.sh

```diff
@@ -4,5 +4,6 @@ set -e -o pipefail
 CERT="${NEW_CERT_FILE:-/var/www/miq/vmdb/certs/server.cer}"
 KEY="${NEW_KEY_FILE:-$CERT.key}"
 if [ ! -f "$CERT" -a ! -f "$KEY" ]; then
-  (umask 077 ; openssl req -x509 -newkey rsa -days 1095 -keyout $KEY -out $CERT -subj "/CN=server" -nodes -batch)
+  openssl req -x509 -newkey rsa -days 1095 -keyout $KEY -out $CERT -subj "/CN=server" -nodes -batch
+  chmod 0660 $CERT $KEY
```

Comment:
Went with `rw-rw----` for both because that's what openssl did for the `server.cer` though I'm not sure that the manageiq group needs write so we might get away with `rw-r-----`